aws/rust-runtime/aws-auth/Cargo.toml

```diff
@@ -19,3 +24,4 @@ http = "0.2.3"
 test-env-log = { version = "0.2.7", features = ["trace"] }
 tokio = { version = "1", features = ["macros", "rt", "rt-multi-thread", "test-util"] }
 tracing-subscriber = { version = "0.2.16", features = ["fmt"] }
+smithy-http = { path = "../../../rust-runtime/smithy-http", features = ["sleep-tokio"] }
```

Comment:
you actually don't want to mention the feature here—mentioning it here makes it a hard dependency

Reply:
when timeout-tokio is enabled, it will get pulled in

Reply:
This should be fine since it's under `[dev-dependencies]`.